<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contacts</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>198</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="category">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>First name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="firstName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Surname</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Street address</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>City</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Phone number</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="surname"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="address"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="city"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="code"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="phone"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteEntry">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>831</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="displayWholeList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Display whole list</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Based on category:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="catToFind">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>226</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="displayTelList">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>223</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Display telephone list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="displayAddrList">
    <property name="geometry">
     <rect>
      <x>542</x>
      <y>224</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Display address list</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>category</tabstop>
  <tabstop>firstName</tabstop>
  <tabstop>surname</tabstop>
  <tabstop>address</tabstop>
  <tabstop>city</tabstop>
  <tabstop>code</tabstop>
  <tabstop>phone</tabstop>
  <tabstop>add</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
